src/main/java/com/google/sps/servlets/ActionsServlet.java

```diff
@@ -82,7 +82,7 @@ private String urlQuery(String basePath, Map<String, String> parameters) throws
   }
 
   /**
-   * Extracts and returns the a list of projects given a jsonStirng.
+   * Extracts and returns a list of projects given a jsonStirng.
```

Comment:
[Nit] String is spelled wrong.

Also, a jsonString is just a string in JSON format right?  In that case I would say "given a JSON string" rather than "given a jsonString" as the latter makes it look like jsonString is some special type of class object (unless it is supposed to be a special type of object in which case I am mistaken)

Reply:
It is a string in JSON format. I'll fix it.